cfgov/regulations3k/tests/test_models.py

```diff
@@ -125,7 +125,7 @@ def test_routable_page_get_context(self):
 
     def test_get_reg_nav_items(self):
         request = HttpRequest()
-        request.url = '/eregulations3k/1002/1002-4/'
+        request.url = '/regulations/1002/1002-4/'
```

Comment:
probably doesn't matter for the test, but we're using the form `/1002/4/` now.

Reply:
Good call